### PR TITLE
[CGKIELE-495][CGKIELE-496][CGKIELE-497] Tackling inconveniences

### PR DESCRIPTION
**Pull request by rtkaczyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,18 +222,13 @@ mallet> getReceipt('0x00039c02c1ca8cb2b25226f74887fc0afcf485797de65afbc105dab134
...
-  contractAddress: null,
-  logs: [],
-  status: '0x00',
-  returnData: '0x' }  
+  ...  
 ```
 
 Note that the receipt may not be readily available, indicated with `null` value, as it takes time for a transaction to be forged.
 
+If the the receipt is for a IELE transaction (technically if it has `statusCode` field and the return data is RLP-decodable) then `returnData` field contains an array of integers, and raw undecoded data (hex string) is available in `rawReturnData`.
```
Comment (LukasGasior1) [inline README.md]: `the the`

File: README.md [hunk 2]
```diff
@@ -338,8 +337,58 @@ mallet> getReceipt()
...
   logs: [],
-  status: '0x00',
-  returnData: '0xc0' }
+  statusCode: '0x00',
+  status: true,
+  returnData: [],
+  rawReturnData: '0xc0' }
+```
+
+#### `iele.constantCall`
```
Comment (LukasGasior1) [inline README.md:346]: why not simply `iele.call`?

File: lib/ieleTranslator.js [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+  return num;
+}
+
+/**
+ * Encode a (Solidity) value to IELE value
+ * @param {string | string[]} value 
+ * @param {{type:string, components?:object[]}} type
+ * @return {string} encoded IELE value
+ */
+function encode(value, type) {
```
Comment (LukasGasior1) [inline lib/ieleTranslator.js:56]: :scream_cat:

File: README.md (hunk 2 above)
Comment (rtkaczyk) [reply README.md:346]: Well... I would be fine with that, as would be people familiar with JSON-RPC/web3. But my understanding is we're also targeting users who aren't

File: lib/ieleTranslator.js (hunk 3 above)
Comment (rtkaczyk) [reply lib/ieleTranslator.js:56]: Wut? 😬

Comment (rtkaczyk) [reply lib/ieleTranslator.js:56]: It is stolen from https://github.com/runtimeverification/remix-ide (which I don't seem to have access to anymore :/ )

Comment (rtkaczyk) [reply lib/ieleTranslator.js:56]: This: https://github.com/input-output-hk/remix-ide/blob/phase/iele_testnet/remix/remix-lib/src/execution/ieleTranslator.js

**Review by LukasGasior1 — APPROVED** [review]

**Review by LukasGasior1 — APPROVED** [review]

**Pull request merged**